fortran/src/H5VLff.F90

```diff
@@ -401,6 +401,41 @@ END FUNCTION H5VLunregister_connector
 
   END SUBROUTINE H5VLunregister_connector_f
 
+!>
+!! \ingroup FH5VL
+!!
+!! \brief Determines whether two connector identifiers refer to the same connector.
+!!
+!! \param conn_id1 A valid identifier of the first connector to check
+!! \param conn_id2 A valid identifier of the second connector to check
+!! \param are_same Whether connector IDs refer to the same connector
+!! \param hdferr    \fortran_error
+!!
+!! See C API: @ref H5VLcmp_connector_cls()
+!!
+  SUBROUTINE H5VLcmp_connector_cls_f(are_same, conn_id1, conn_id2, hdferr)
```

Comment:
This routine is necessary because it's possible to have VOL connector IDs that are different, but point to the same underlying VOL connector class.